spec/ip_address_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'ip_ranger'
+
+RSpec.describe IPRanger::IPAddress do
```

Comment:
Can we move this file to `spec/ip_ranger/ip_address_spec.rb` to mirror the layout of `lib`?

I also notice that I missed renaming `spec/russet_spec.rb` to `spec/ip_ranger_spec.rb` but maybe that'd be better as `spec/ip_ranger/ip_range_spec.rb` anyway.